nixos/modules/tasks/network-interfaces.nix

```diff
@@ -487,9 +513,8 @@ in
 
           interfaces = mkOption {
             example = [ "eth0" "eth1" ];
```

Comment:
Are you sure the example still compatible with the new type?

Reply:
Good catch! I forgot to update the example.